dockers/docker-dhcp-server/cli/config/plugins/dhcp_server.py

```diff
@@ -1,13 +1,371 @@
+    elif ip_list:
+        dbconn.hmset("CONFIG_DB", key, {"ips": ",".join(ip_list)})
+    else:
+        raise Exception("One of range and ip_list should be provided")
+
+
+@ip.command()
+@click.argument("vlan_interface", required=True)
+@click.argument("interface", required=True)
+@click.options("--range", required=False, nargs=-1)
+@click.argument("ip_list", required=False, nargs=-1)
+@clicommon.pass_db
+def unbind(db, vlan_interface, interface, range_, ip_list):
+    pass
+
+
+@ipv6.group(cls=clicommon.AliasedGroup)
+def ip():
+    pass
+
+
+@ip.command()
+@click.argument("vlan_interface", required=True)
+@click.argument("interface", required=True)
+@click.options("--range", required=False, nargs=-1)
+@click.argument("ip_list", required=False, nargs=-1)
+@clicommon.pass_db
+def bind(db, vlan_interface, interface, range_, ip_list):
+    pass
+
+
+@ip.command()
+@click.argument("vlan_interface", required=True)
+@click.argument("interface", required=True)
+@click.options("--range", required=False, nargs=-1)
+@click.argument("ip_list", required=False, nargs=-1)
+@clicommon.pass_db
+def unbind(db, vlan_interface, interface, range_, ip_list):
+    pass
+
+
+@ipv4.group(cls=clicommon.AliasedGroup)
+def option():
+    pass
+
+
+@option.command()
+@click.argument("option_name", required=True)
+@click.argument("option_id", required=True)
+@click.argument("type", required=True)
+@click.argument("value", required=True)
+@clicommon.pass_db
+def add(db, option_name, option_id, type_, value):
+    assert option_id.isdigit()
+    assert type_ == "string"
+    dbconn = db.db
+    key = "DHCP_SERVER_IPV4_CUSTOMIZED_OPTIONS|" + option_name
+    dbconn.hmset("CONFIG_DB", key, {
+        "option_id": option_id,
+        "type": type_,
+        "value": value,
+        })
+
+
+@option.command()
+@click.argument("option_name", required=True)
+@clicommon.pass_db
+def del(db, option_name):
+    dbconn = db.db
+    key = "DHCP_SERVER_IPV4_CUSTOMIZED_OPTIONS|" + option_name
+    dbconn.delete("CONFIG_DB", key)
+
+
+@option.command()
+@click.argument("dhcp_interface", required=True)
+@click.argument("option_list", nargs=-1)
+@clicommon.pass_db
+def bind(db, dhcp_interface, option_list):
+    dbconn = db.db
+    key = "DHCP_SERVER_IPV4|" + dhcp_interface
+    if dbconn.exists("CONFIG_DB", key) and option_list:
+        old_value = dbconn.get("CONFIG_DB", key, "customized_options")
+        if old_value:
+            option_list = [old_value] + option_list
+        dbconn.set("CONFIG_DB", key, "customized_options", ",".join(option_list))
+
+
+@option.command()
+@click.argument("dhcp_interface", required=True)
+@click.argument("option_list", nargs=-1)
+@click.option("--all", required=False, default=False, is_flag=True)
+@clicommon.pass_db
+def unbind(db, dhcp_interface, option_list, all_):
+    dbconn = db.db
+    key = "DHCP_SERVER_IPV4|" + dhcp_interface
+    if all_:
+        dbconn.set("CONFIG_DB", key, "customized_options", "")
```

Comment:
In yang model, "customized_options" field is not required. I think maybe we can delete this field if it is empty?